cmd/controller/main.go

```diff
@@ -77,9 +78,9 @@ func main() {
 		logger.Fatalw("Error building kubeconfig", zap.Error(err))
 	}
 
-	// We run 6 controllers, so bump the defaults.
-	cfg.QPS = 6 * rest.DefaultQPS
-	cfg.Burst = 6 * rest.DefaultBurst
+	const numControllers = 7
```

Comment:
You and @ZhiminXiang both bump this.  I like the constant, can we check that it matches `len(controllers)` and `logger.Fatalf` below, so that folks can't check in without adjusting this multiplier?